gravitee-apim-portal-webui-next/src/components/subscription/webhook/consumer-configuration/consumer-configuration.component.ts

```diff
@@ -14,34 +14,30 @@
  * limitations under the License.
  */
 import { AsyncPipe } from '@angular/common';
-import { Component, DestroyRef, inject, Input, OnInit } from '@angular/core';
-import { takeUntilDestroyed } from '@angular/core/rxjs-interop';
+import { Component, EventEmitter, input, Input, InputSignal, OnInit, Output } from '@angular/core';
 import { FormControl, FormGroup, ReactiveFormsModule, Validators } from '@angular/forms';
 import { MatButton } from '@angular/material/button';
 import { MatCard, MatCardContent, MatCardHeader } from '@angular/material/card';
-import { MatDialog } from '@angular/material/dialog';
 import { MatFormFieldModule, MatLabel } from '@angular/material/form-field';
 import { MatIcon } from '@angular/material/icon';
 import { MatInputModule } from '@angular/material/input';
 import { RouterLink } from '@angular/router';
-import { catchError, combineLatest, map, Observable, startWith, switchMap, tap } from 'rxjs';
+import { map, Observable, startWith } from 'rxjs';
 import { of } from 'rxjs/internal/observable/of';
 
 import { ConsumerConfigurationForm, ConsumerConfigurationValues } from './consumer-configuration.models';
-import { Plan } from '../../../../entities/plan/plan';
-import { Subscription, SubscriptionConsumerConfiguration, UpdateSubscription } from '../../../../entities/subscription';
-import { PlanService } from '../../../../services/plan.service';
-import { SubscriptionService } from '../../../../services/subscription.service';
+import { Subscription } from '../../../../entities/subscription';
 import { deepEqualIgnoreOrder } from '../../../../utils/deep-equal-ignore-order';
-import {
-  SubscriptionCommentDialogComponent,
-  SubscriptionCommentDialogData,
-} from '../../subscription-comment-dialog/subscription-comment-dialog.component';
 import { ConsumerConfigurationAuthenticationComponent } from '../consumer-configuration-authentification';
 import { ConsumerConfigurationHeadersComponent } from '../consumer-configuration-headers';
 import { ConsumerConfigurationRetryComponent } from '../consumer-configuration-retry';
 import { ConsumerConfigurationSslComponent } from '../consumer-configuration-ssl';
 
+export interface ConsumerConfigurationFormData {
```

Comment:
there is a model file in this directory I think if you want to move the interface in it